deployment/ccip/changeset/v1_5/cs_jobspec.go

```diff
@@ -89,10 +85,17 @@ func JobSpecsForLanesChangeset(env deployment.Environment, c JobSpecsForLanesCon
 				// This is so that we can retry the proposal with manual intervention.
 				// JOBID will be empty if the proposal failed.
 				return deployment.ChangesetOutput{
-					Jobs: Jobs,
+					Jobs: append(Jobs, deployment.ProposedJob{
+						Node: nodeID,
+						Spec: job,
+					}),
```

Comment:
Thanks for opening this. I forgot to update this in this spec. @makramkd recommended returning empty value in case of err. I updated it in other jobspec CS but missed to incorporate that review comment here.